src/test/kotlin/com/tylerthrailkill/helpers/prettyprint/TestSetup.kt

```diff
@@ -18,12 +18,12 @@ fun Root.setup() {
  * Test helper function, defaults to providing no tab size and wrapping the print stream in
  * a ByteArrayOutputStream to test against
  */
-fun TestBody.prettyPrint(obj: Any?, tabSize: Int? = null, wrappedLineWidth: Int? = null): ByteArrayOutputStream {
+fun TestBody.prettyPrint(obj: Any?, tabSize: Int? = null, wrappedLineWidth: Int = 80): ByteArrayOutputStream {
```

Comment:
could you please leave this as `? = null`. 

I did that on purpose as there is a difference between running a test and using a default (in the test) and using the default from the class itself.

Reply:
I think this is functionally equivalent to what you had before, but I'll do it if you think it makes more sense semantically

Reply:
Technically it is functionally equivalent, but if you were to change the default break width in `PrettyPrint.kt`, then this setup would continue to pass 80 to each test. If you had a test that tested whether the line broke at 80 or not then it would continue to pass, even if though the test should technically be failing